inception/inception/BUILD

```diff
@@ -124,6 +135,17 @@ py_library(
     ],
 )
 
+py_library(
+    name = "inception_distributed_train",
+    srcs = [
+        "inception_distributed_train.py",
+    ],
+    deps = [
```

Comment:
I think this applies to all of the build rules in here. Perhaps do in a follow-up PR? (I'm not sure what that option actually makes....)